testsuites/syncgateway/functional/2sg_1cbs/test_sg_replicate.py

```diff
@@ -185,9 +252,20 @@ def test_sg_replicate_continuous_replication():
     assert_does_not_have_doc(sg2_user, doc_id_3)
 
 
-def test_sg_replicate_delete_db_replication_in_progress():
+@pytest.mark.sanity
+@pytest.mark.syncgateway
+@pytest.mark.sgreplicate
+@pytest.mark.usefixtures("setup_2sg_1cbs_suite")
+def test_sg_replicate_delete_db_replication_in_progress(setup_2sg_1cbs_test):
```

Comment:
is there a pytest decorator to skip this test since it's failing due to a known issue?  or should we comment it with a link to the issue?